### DISCUSSION TITLE
Can you add a hide category please so I can hide my NSFW stuff while keeping everthing else the same

### DISCUSSION THREAD

**Discussion by kalitbartman** [xgi/houdoku#344] [Feature Requests]
Can you add a hide category so I can filter some of my added categories out please as I would like to hide my NSFW stuff so I don't need to add all my SFW manga's to a category thanks.

**Comment by kalitbartman** [comment]
so far I have to use a main and NSFW category I put adult stuff into NSFW and everthing else into main and select main to hide NSFW so I would need to add them to main category every time I add a new manga which is tedious

**Reply by WilliShakes** [reply]
One idea would be to add the option to select a category when adding series. Even better would be a category filter so we can toggle our NSFW stuff or anything else at will.
